invenio_communities/assets/semantic-ui/js/invenio_communities/members/searchBarWithFilters.js

```diff
@@ -7,9 +7,9 @@
  */
 
 import React, { Component } from "react";
-import { Grid, Label, Dropdown, Button, Icon } from "semantic-ui-react";
+import { Grid, Label, Dropdown, Button, Icon, Popup } from "semantic-ui-react";
```

Comment:
could you change the file name to start with a capital letter? It would be consistent with the rest of the components

Reply:
Right! Thanks!